composer.json

```diff
@@ -13,11 +13,13 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0",
-        "illuminate/support": "~5.0|^6.0|^7.0|^8.0|^9.0|^10.0"
+        "php": ">=8.1",
+        "illuminate/support": "~5.0|^6.0|^7.0|^8.0|^9.0|^10.0",
+        "laravel/framework": "^8.0|^9.0|^10.0"
```

Comment:
Looks good, just here it should be possible to not import the whole `laravel/framework`, maybe just by not using the facade for the session?

Reply:
Unfortunately, `laravel/framework`  is required because of `base_path`  helpers used in the project.

Definition of the helper is here: https://github.com/laravel/framework/blob/10.x/src/Illuminate/Foundation/helpers.php